addon/components/password-toggle.js

```diff
@@ -2,20 +2,17 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
     didInsertElement: function() {
-        $('button').on('click', function(){
-          var text = $(this).text();
-          var $parent = $(this).parent();
-          var $input = $('input', $parent).clone();
+        $('button').on('click', function() {
+            var text = $(this).text();
+            var $input = $(this).parent().find('input');
 
-          $('input', $parent).remove();
```

Comment:
I remember talking with Nick about why we decided to clone instead of hide/show and it had something to do with browser support. Nick -can you confirm this non clone version (the one in this PR) will work on IE9/10, Firefox, Chrome, iOS 8 Safari before we merge it?